setup.cfg

```diff
@@ -2,11 +2,13 @@
 author = rioxarray Contributors
 author_email = alansnow21@gmail.com
 name = rioxarray
+version = attr: rioxarray._version.__version__
```

Comment:
Due to importing methods to the main `rioxarray` namespace, this is not possible as it calls external dependencies potentially not installed yet. It will have to be reverted to the previous method for retrieving `__init__.py`.

Reply:
This normally isn't an issue with `python -m build` (#536) which uses a venv isolated environment by default to gather the specified dependencies from `install_requires`. The issue you are describing can only be triggered using `python -m build --no-isolation` to show (e.g.) `ModuleNotFoundError: No module named 'numpy'`.

Reply:
I wonder if this is the line causing the troubles in the docker build: https://github.com/corteva/rioxarray/blob/39c918930ed2694e979cd557556f9a28dc296ae4/.github/workflows/tests.yaml#L80

Reply:
hmm, perhaps I'll need to investigate this a bit further...

Reply:
yes, I was able to reproduce this, but essentially all "python setup.py" commands are more-or-less deprecated and should be phased out (e.g. [here](https://stackoverflow.com/questions/72468946/setuptools-pyproject-toml-equivalent-to-python-setup-py-clean-all)). Was there a specific reason for it?

Reply:
Sounds like it isn't necessary, so you can remove it.